en/components/grid.md

```diff
@@ -22,345 +22,503 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 
 ### Dependencies
 
-The grid is exported as as an `NgModule`, thus all you need to do in your application is to import the _IgxGridModule_
-inside your `AppModule`:
+The grid is exported as as an `NgModule`, thus all you need to do in your application is to import the _IgxGridModule_ inside your `AppModule`:
 
-```typescript
-// app.module.ts
+    // app.module.ts
 
-import { IgxGridModule } from 'igniteui-angular/main';
+    import { IgxGridModule } from 'igniteui-angular/main';
+    // Or
+    import { IgxGridModule } from 'igniteui-angular/grid';
 
-@NgModule({
-    imports: [
-        ...
-        IgxGridModule,
-        ...
-    ]
-})
-export class AppModule {}
-```
+    @NgModule({
+        imports: [
+            ...
+            IgxGridModule.forRoot(),
+            ...
+        ]
+    })
+    export class AppModule {}
 
-Each of the components, directives and helper classes in the _IgxGridModule_ can be imported either through the _grid.component_ or through
-the main bundle in _igniteui-angular_. While you don't need to import all of them to instantiate and use the grid, you usually will import them
-(or your editor will auto-import them for you) when declaring types that are part of the grid API.
+Each of the components, directives and helper classes in the _IgxGridModule_ can be imported either through the _grid_ sub-package or through the main bundle in _igniteui-angular_. While you don't need to import all of them to instantiate and use the grid, you usually will import them (or your editor will auto-import them for you) when declaring types that are part of the grid API.
 
-```typescript
-import { IgxGridComponent } from 'igniteui-angular/grid/grid.component';
-...
+    import { IgxGridComponent } from 'igniteui-angular/grid/';
```

Comment:
Why were the code delimiters removed?  In this way, you won't get syntax highlighting, right?